### DISCUSSION TITLE
Primeiro emprego

### DISCUSSION THREAD

**Discussion by dericparra** [frontendbr/forum#1682] [Discussão]
Como conseguir o primeiro emprego sem "experiencia" morando longe da capital ? (Após bastante estudo e os primeiros projetos pessoais)

**Comment by FabricioBernardes** [comment]
Também estou tentando descobrir :/

**Comment by guilhermesantoss** [comment]
Uma maneira de conseguir o primeiro emprego mesmo sem experiência e morando longe, é o "Network", por exemplo essa discussão no GitHub, uma ótima oportunidade pra trocar figurinha com os participantes da comunidade.

**Comment by dericparra** [comment]
A minha intenção inicial foi quase essa mesmo Guilerme, interagir e conhecer mais devs com quem posso aprender ou ensinar algo

**Comment by italosilva01-zz** [comment]
Alguém sabe um material bom de javascript e que der resultado rápido? Estava estudando pelo livro eloquente js, mas o ritmo do livro é bem devagar. Queria aprender coisas mais funcionais o quanto antes

**Comment by FabricioBernardes** [comment]
Hahahaha

Acho que todos viemos aqui tentando nos integrar.

Bem, em relação ao aprendizado, me identifiquei muito com o [CFB Cursos](https://www.youtube.com/user/canalfessorbruno). O material dele é bastante completo. Cada vez mais também estou consumindo conteúdo do [FreeCodeCamp](https://www.youtube.com/channel/UC8butISFwT-Wl7EV0hUK0BQ), que tem vídeos grandes tratando de partes especificas da temática. Em termos de conteúdo "prático" eu gosto de usar os códigos do [HowTo da W3](https://www.w3schools.com/howto/default.asp) como referência para fazer pequenas aplicações, sempre aplicando algumas modificações.

E bora conversar gente :)

**Comment by guilhermesantoss** [comment]
Um site muito bom para aprender Javascript é o da [RocketSeat](https://rocketseat.com.br), que nos últimos anos inovaram totalmente o jeito de ensinar essa linguagem por meio de vídeos, desafios e atividades didáticas e com uma riqueza de informações que se enquadram muitas vezes com problemas reais.

**Comment by FabricioBernardes** [comment]
Verdade, aprendi muita coisa na ultima Semana Omnistack.

**Comment by andrebonfimdev** [comment]
> Um site muito bom para aprender Javascript é o da [RocketSeat](https://rocketseat.com.br), que nos últimos anos inovaram totalmente o jeito de ensinar essa linguagem por meio de vídeos, desafios e atividades didáticas e com uma riqueza de informações que se enquadram muitas vezes com problemas reais.

Verdade, excelente didática. Comecei agora o **Launchbase**(para iniciantes) e quero o quanto antes dar continuidade e fazer o **Gostack**(avançado). Claro que já fiz vários cursos, mas esse é o único que abriu totalmente minha mente. Te joga direto para o dia a dia do Dev. É como se você realmente estivesse trabalhando em um projeto. Além da comunidade no **Discord** ser incrível. Estou extretamente feliz com a **Rocketseat** e posso garantir com propriedade que vou sair um Dev pronto para o mercado de trabalho. Recomendo!! #1512

**Comment by sbrj** [comment]
> 
> 
> Uma maneira de conseguir o primeiro emprego mesmo sem experiência e morando longe, é o "Network", por exemplo essa discussão no GitHub, uma ótima oportunidade pra trocar figurinha com os participantes da comunidade.

Uma maneira interessante é se deixar ser encontrado, como diz o  Filipe Deschamps
 em um vídeo.... e faz bastante sentido. Ele informa que em determinado momento tinha que contratar vários programadores e fez um script para montar a lista dele. Ele fala sobre estar aqui no github, por exemplo. Então seria algo assim: estar no máximo de lugares possíveis, fazendo o que puder de interessante para deixar a sua marca de alguma forma. Concordam?

**Comment by leandroruel** [comment]
eu moro longe, e a receita para arrumar o primeiro emprego que eu pensei na época é: "se eu começar em empresas pequenas, com poucos funcionários, a chance é maior"... na primeira entrevista fui contratado numa agência, agências são a porta de entrada de muito dev. mas não é para ficar para sempre rs.

**Comment by italosilva01-zz** [comment]
> 
> 
> eu moro longe, e a receita para arrumar o primeiro emprego que eu pensei na época é: "se eu começar em empresas pequenas, com poucos funcionários, a chance é maior"... na primeira entrevista fui contratado numa agência, agências são a porta de entrada de muito dev. mas não é para ficar para sempre rs.

Como você se comportou na entrevista ? Tem alguma dicas ai. Comecei a procurar jobs na área agora, estou atrás de um estágio.

**Comment by leandroruel** [comment]
é só dizer o que você sabe eo que não sabe com bastante confiança, a quanto tempo mexe em cada ferramenta/framework/linguagem e tá tudo certo.

**Comment by LuanDc** [comment]
Tenta desenvolver alguns projetos próprios, criar um repositório aqui no Github e participar das comunidades pra fazer networking. Isso vai te abrir portas pra trabalhos remotos ou possivelmente propostas vindo dos grandes centros.

**Comment by felquis** [comment]
Foram muitas postagens em um blog e compartilhar os links em grupos de discussão pedindo feedback que me deram o contato para o primeiro emprego na área.

**Comment by lddv** [comment]
@italosilva01 além do que o @leandroruel disse, acredito que vale mostrar interesse em aprender coisas novas e não ter medo de atacar o problema.

Muitas vezes ter bom ânimo em procurar a solução é mais importante, principalmente no início, do que de fato saber qual é a resposta do problema. São essas experimentações que vão te dar a base pra escolher a melhor solução.

**Comment by ninetails** [comment]
Minha opinião de *$#@ mas sendo meio realista, não tomem como verdade.

Existe trabalho remoto, mas estes vão sempre querer os melhores profissionais que já saibam trabalhar sem ter alguém pra ir na tua mesa e falar pra vc o que e como vc tem que fazer. Ou seja, não é impossível mas é difícil e pouco provável conseguir um trabalho desses logo de começo.

Se a tua cidade possui universidade com cursos voltados a tecnologia, facilita empregabilidade pois podem ter empresas e startups ligadas a esse meio.

Mas curti bem o que a galera discutiu aqui: atinja visibilidade. Esteja no GitHub e principalmente no LinkedIn. Aumente sua participação nessas redes pois se não for seu intuito sair do lugar de origem no momento, é onde podem tentar te achar, LinkedIn principalmente.

Falando necessariamente na estratégia de começar de onde vc reside e não se arriscar a ir a grandes polos: procure empresas perto de ti, e lembre que como vc tá começando, nem tudo pode ser perfeito.... Provavelmente vc irá utilizar de códigos legados, linguagens fora da moda, mas irá fazer produtos, irá começar a aprender programação na prática e vai começar a ganhar sua própria renda.

Disso, tendo seu começo, guarde e faça isso seu capital e invista na sua carreira: cursos e principalmente eventos de alguma cidade perto ou grandes centros, para ampliar seu networking.

Paralelamente desde cedo vc tb pode tentar fazer networking ajudando em algum projeto no GitHub.

E uma dica: Experiência no começo conta muito como diferencial pq RH e empresas são loucas, querendo experiência de galera que nunca trabalhou. Portanto faça experimentações de seus estudos e publique! Se venda. É aceitável vc falar pro RH que não teve experiência só que "olha o aplicativo que eu fiz quando tava estudando *xxxxxx*" e tem interesse em aprender.
